lib/improver/utilities/cube_manipulation.py

```diff
@@ -853,9 +874,9 @@ def enforce_coordinate_ordering(
     # Transpose by inserting the requested coordinates at either the start
     # or the end.
     if anchor == "start":
-        cube.transpose(coord_dims+remaining_coords)
+        cube.transpose(coord_dims + remaining_coords)
     elif anchor == "end":
-        cube.transpose(remaining_coords+coord_dims)
+        cube.transpose(remaining_coords + coord_dims)
```

Comment:
Not sure whether this is right now or was right before.  As a function argument, I think it was right before.  However I'm not sure whether PEP8 is prescriptive on this: https://www.python.org/dev/peps/pep-0008/#other-recommendations

Reply:
I must admit I thought in this case for PEP8 you should have a space either side of the `+` . It certainly looks clearer and is easier to read. This was due to PyCharm's PEP8 formatting and its usually very good. I can see what the other reviewer thinks.....

Reply:
Fair enough!